modules/jaggery-apps/portal/gadgets/user_profile/controllers/my-profile/edit-finish.jag

```diff
@@ -1,4 +1,19 @@
 <%
+/*
+ * Copyright (c) WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Year has been added in following PR
https://github.com/wso2/product-is/pull/571